dashboard/test/lib/queue_processor_test.rb

```diff
@@ -155,7 +154,7 @@ def test_queue_processor
     # Wait for the queue processor to handle all of the messages.
     while handler.received_bodies.proper_subset?(expected_bodies)
       processor.assert_workers_alive_for_test
-      sleep 2
+      sleep 1
```

Comment:
As the entire test takes roughly three seconds, experimentally this change reduces the running time of the test by one second (previously roughly four seconds).

Reply:
If the loop condition is never met, this will loop forever. We should probably add a timeout and fail.

Reply:
Agreed. That said, I'd prefer to do so in another PR, I want to minimize the chance we introduce mechanisms of failure while reenabling the test. In particular, I don't know if runs on CircleCI and/or test have comparable durations to local runs.